modules/ROOT/partials/import-database.adoc

```diff
@@ -39,7 +39,7 @@ If the database you want to upload is running an earlier version of Neo4j, pleas
 
 [CAUTION]
 ====
-The `neo4j-admin push-to-cloud` command in Neo4j 4.4 and earlier is not compatible with instances encrypted with xref:platform/security.adoc#_customer_managed_keys[Customer Managed Keys].
+The `neo4j-admin push-to-cloud` command in Neo4j 4.4 and earlier is not compatible with instances encrypted with xref:platform/security/encryption-at-rest.adoc#_customer_managed_keys[Customer Managed Keys].
```

Comment:
You may want to consider renaming this file to just _encryption.adoc_.

Reply:
I've renamed it!